settings/htsohm.sample.yaml

```diff
@@ -36,15 +36,15 @@ lattice_constant_limits: [25.6, 51.2]
 number_density_limits: [1.49e-05, 0.02122]
 sigma_limits: [1.052, 6.549]
 
-methane_loading_limits: [0, 350]
+gas_loading_limits: [0, 350]
 surface_area_limits: [0, 4500]
 void_fraction_limits: [0, 1]
 
 retests:
   number: 3
   tolerance: 0.25
 
-methane_loading:
+gas_loading:
   simulation_cycles: 100
   initialization_cycles: 100
 helium_void_fraction:
```

Comment:
Great separation of one thing into one commit. It's obvious what you did (though it probably should have been named something like: "generalize methane_loading to gas_loading so we can test different gases without the variable names being confusing") and there are a lot of changes but because they are grouped, I was able to just scan through the whole thing super fast. Good!